packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -94,25 +87,39 @@ const ShowHide = (props) => {
 
   const theme = React.useContext(ThemeContext);
   const contentRef = React.useRef(null);
+  const [activeElement, setActiveElement] = React.useState(null);
```

Comment:
This state variable is only for managing the state of the contentRef tab index.  It should be a Boolean value, right?

Reply:
My idea was to store an actual active element: contentRef element, focusRef element or null, if there is no active element at the moment. Boolean option has only 2 options and it is not enough for me to make a proper decision, as false would only indicate that contentRef is not an active element, saying nothing of if focusRef is currently active or there is no active element at all. I need to know an actual active element for blur, for setting tabindex, and it needs to have 3 options: contentRef element, focusRef element or no element at all, that is why boolean wouldn't work.

Reply:
Your state variable is only controlling whether we are setting the tabIndex on the content ref.  It is causing you to add unnecessary logic, as-is.

Reply:
It's not an unnecessary logic, it is the same logic that we need in case you suggested, just keeping track on active element for blur event, 3 states instead of 2, not much different. I changed it to look the way you want in this commit to resolve it, but it is pretty much the same: https://github.com/cerner/terra-core/pull/3838/commits/170f7cbd0b5ff79272fb22d69c3b7d268398c352